core/src/Revolution/modUser.php

```diff
@@ -400,7 +400,7 @@ public function addSessionContext($context)
                     $ua->set('logincount', $ua->logincount + 1);
                     $ua->set('lastlogin', $ua->thislogin);
                     $ua->set('thislogin', time());
-                    $ua->set('sessionid', session_id());
+                    /*$ua->set('sessionid', session_id());*/
```

Comment:
Consider either just removing this line instead of commenting it out, or add an extra comment noting the intent.